src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,4 +121,28 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Simple check that validates label is not empty string after trimming.
+     */
+    public static String parseLabel(String value) throws ParseException {
+        requireNonNull(value);
+        String trimmedValue = value.trim();
+        if (trimmedValue.equals("")) {
```

Comment:
Is it better to have a method to test the validity of the trimmedValue (e.g. isValidLabel)?